core/src/main/scala/org/apache/spark/network/Security.scala

```diff
@@ -35,33 +37,15 @@ object Security extends Logging {
     }
   }
 
-  def enableSSL(spark: SparkSession, conf: H2OConf): Unit = enableSSL(spark, conf.sparkConf)
+  def enableSSL(spark: SparkSession, conf: H2OConf): Unit = enableSSL(conf)
```

Comment:
Shouldn't we deprecate this method overloading?

Reply:
Yup, I've done it in a separate PR, will rebate this one onto it, thanks